### DISCUSSION TITLE
fabric api is dowloading as a document

### DISCUSSION THREAD

**Discussion by catcantgame** [FabricMC/fabric-api#1451] [Player support] [answered]
i tried to dowload fabric api but its dowloading as a document not as its sopposed to, i have java and i also dowloaded fabric and it works, only mods and fabric api are dowloading as files and i dont know what to do i even reinstalled java but it didnt help.
![20210511_123252](https://user-images.githubusercontent.com/84012948/117793939-6a480080-b255-11eb-8c6e-a2fb4a89feb9.jpg)

![20210511_123405](https://user-images.githubusercontent.com/84012948/117793961-6e741e00-b255-11eb-906b-5add8227c643.jpg)

**Comment by haykam821** [comment, marked as answer]
The file association does not matter as long as it has the proper extension (`.jar`) and is placed in the `mods` directory.
